internal/host/plugin/repository_host_catalog.go

```diff
@@ -512,10 +529,6 @@ func (r *Repository) DeleteCatalog(ctx context.Context, id string, _ ...Option)
 		return db.NoRowsAffected, errors.Wrap(ctx, err, op)
 	}
 	sets, _, err := r.getSets(ctx, "", c.GetPublicId())
-	if err != nil {
```

Comment:
Was this on purpose? Do we not want to check the error? If not should replace with `_`.

Reply:
Woops.  I started this branch based off of an unfinished branch where the update functionality was being added to the repo.  I hadn't updated this branch with those changes yet.  I've done so now.